### PR TITLE
Fix for yarn start on Windows

### PR DESCRIPTION
**Pull request by niemam29**
# Summary

Noticed that on Windows there are some problems with starting swapr due to env variables. Fixed that by cross-env-shell

  # To Test

1. Try to run yarn start

### CODE REVIEW

**Comment by wixzi** [comment]
I think this will break in linux and mac environments. Also if your commit message is same try to `amend commit` and  ` force push` so it don't create so many commits

**Comment by denciu** [comment]
> I think this will break in linux and mac environments. Also if your commit message is same try to `amend commit` and ` force push` so it don't create so many commits

Works on ubuntu

**Comment by denciu** [comment]
> I think this will break in linux and mac environments. Also if your commit message is same try to `amend commit` and ` force push` so it don't create so many commits

That's true, on ubuntu it generates files under completely different directory. That's also a reason why CI fails. See img:
image

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
...
     "synpress:run": "cross-env MOCHAWESOME_HTML='false' MOCHAWESOME_OVERWRITE='false' NETWORK_NAME='rinkeby' CYPRESS_SKIP_RESOURCES_WAIT='true' FAIL_ON_ERROR='0' PRIVATE_KEY='367f6baeb27b33408c85040e13d1ba3e2a173a9112eb3b103f79df0729ddda72' ETHERSCAN_KEY='25VF97IQZ8EFIYSYFI72UCTJJII3HSZU7Z' synpress run --configFile tests/synpress/synpress.json --config supportFile='tests/support/index.js',pluginsFile='tests/plugins/index.js' -r mochawesome --reporterOptions 'overwrite=false, html=false, json=true'",
-    "codegen:socket": "TARGET=src/services/EcoBridge/Socket/api/generated && openapi-generator-cli generate --skip-validate-spec -i ./src/services/EcoBridge/Socket/api/swagger.json -g typescript-fetch -o \"$TARGET\" --additional-properties=typescriptThreePlus=true",
+    "codegen:execute": "cross-env openapi-generator-cli generate --skip-validate-spec -i ./src/services/EcoBridge/Socket/api/swagger.json -g typescript-fetch -o $TARGET --additional-properties $PROPERTIES",
+    "codegen:socket": "cross-env TARGET=src/services/EcoBridge/Socket/api/generated PROPERTIES='typescriptThreePlus=true' yarn codegen:execute",
```
Comment (adamazad) [inline package.json:21]: I would put all this into a bash. :) cc @denciu

**Review by adamazad — CHANGES_REQUESTED** [review]

**Comment by wixzi** [comment]
> @wixzi @denciu Fixed problems, now it works both on Windows and Linux, would be good to have it tested on Mac. About the commits, they will be squashed before merge anyway because i'm using "squash and merge" from github, so i'm not sure if that's necessary

Its working in mac as well. 👍🏼

**Comment by adamazad** [comment]
@niemam29 fix conflicts and merge

**Review by wixzi — APPROVED** [review]

**Review by berteotti — APPROVED** [review]

**Comment by adamazad** [comment]
@niemam29 is this still an issue?

**Pull request closed without merging**